snafu/cyclictest_wrapper/trigger_cyclictest.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import os
+import re
+import subprocess
+import datetime
+import logging
+
+logger = logging.getLogger("snafu")
+
+
+class Trigger_cyclictest():
+    def __init__(self, args):
+        self.uuid = args.uuid
+        self.user = args.user       
+        self.path = args.path
+        self.samples = args.samples
+
+        self.stressng = args.stressng
+        self.duration = args.duration
+        self.disable_cpu_balance = args.disable_cpu_balance
```

Comment:
These variables are not used anywhere

Reply:
Ah, I should probably add these to the JSON payload.

Reply:
Done